packages/ng/gauge/gauge.component.html

```diff
@@ -0,0 +1,18 @@
+@if (circular()) {
+	<svg
+		class="gauge"
+		[class]="paletteClass"
+		[class.mod-thin]="thin()"
+		[class.is-zero]="value() === 0"
+		[attr.width]="size()"
+		[attr.height]="size()"
+		[attr.viewBox]="`0 0 ${size()} ${size()}`"
+		[attr.style]="`--components-gauge-value: ${value()}; --components-gauge-circleR: ${(size() - thickness()) / 2}px`"
```

Comment:
Does it  handle float values? Would be needed to specify it in docs if that's not the case (example: 33.33)

Reply:
It's taken into account. :)